app/services/group/v1x1_seed.rb

```diff
@@ -0,0 +1,74 @@
+module Group
+  class V1x1Seed
+    attr_reader :status
+    CATALOG_ADMINISTRATOR_GROUP = "Catalog Administrators".freeze
+
+    def initialize
+      @request = Insights::API::Common::Request.current
+      raise Catalog::NotAuthorized unless @request.user.org_admin?
+    end
+
+    def tenant
```

Comment:
I do not see this called but assume it was for `run_seeding` and `mark_as_seeded` below.  Is it needed?